libs/orders/src/lib/components/order-feedback/order-feedback.tsx

```diff
@@ -113,6 +74,27 @@ export const OrderFeedback = ({ transaction, order }: OrderFeedbackProps) => {
           </div>
         )}
       </div>
+      {orderRejectionReason && (
+        <div>
+          <p className={labelClass}>{t(`Reason`)}</p>
+          <p data-testid="error-reason">{t(orderRejectionReason)}</p>
+        </div>
+      )}
     </div>
   );
 };
+
+const rejectionReason = (
+  order: OrderEvent_busEvents_event_Order
+): string | null => {
+  switch (order.status) {
+    case OrderStatus.Stopped:
+      return t(
+        `The network could not fill the ${order.timeInForce} order and it has been stopped`
```

Comment:
suggestion (non-blocking): What about the below, which I think sounds friendlier. 

```suggestion
        `Your ${order.timeInForce} order could not be filled and it has been stopped`
```